apollo-router/src/services/subgraph_service.rs

```diff
@@ -431,6 +431,13 @@ async fn call_websocket(
     subgraph_cfg: &WebSocketConfiguration,
     subscription_hash: String,
 ) -> Result<SubgraphResponse, BoxError> {
+    let operation_name = request
+        .subgraph_request
+        .body()
+        .operation_name
```

Comment:
is that really the expected behaviour here? The operation name that was set in the subgraph request is defined by query planner, so it will be different from the operation name extracted in query analysis

Reply:
oh I see, that was introduced in https://github.com/apollographql/router/commit/ed6a9fb94b024f8144942cecf4ccc3c4ab66f902#diff-6ef5a208ca8622f30eef88f75c18566e0304d59856b66293dcd6811555e6382e